### PR TITLE
[DO-1105] United Register and Dataset

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: data_ocean/fixtures/register.json [hunk 1]
```diff
@@ -8,269 +8,115 @@
...
-    "fields": {
-      "created_at": "2020-09-10 15:40",
-      "name": "PEP",
-      "endpoint": "/api/pep/{id}",
-      "type": "retrieve",
-      "register_id": 1
+      "source_url_address": "https://pep.org.ua/uk/data",
+      "source_api_address": "https://pep.org.ua/opendata/persons/json",
+      "api_list": "/api/pep",
+      "api_detail": "/api/pep/{id}"
```
Comment (roman-tiukh) [inline data_ocean/fixtures/register.json:14]: it is good practice to put a slash at the end of the url
```suggestion
      "api_list": "/api/pep/",
      "api_detail": "/api/pep/{id}/"
```

**Review by roman-tiukh — CHANGES_REQUESTED** [review]
need working views:
1. `api/company/<id>/`
2. `api/company/`
3. `api/company/<source>/<id>/` - retrieve company with filter by source
4. `api/company/<source>/` - list by source

File: data_ocean/fixtures/register.json (hunk 1 above)
Comment (IegorSoboliev) [reply data_ocean/fixtures/register.json]: done

File: data_ocean/fixtures/register.json [hunk 2]
```diff
@@ -115,8 +115,8 @@
       "source_register_id": "",
       "source_url_address": "http://download.companieshouse.gov.uk/en_output.html",
       "source_api_address": "",
-      "api_list": "/api/company/uk",
-      "api_detail": "/api/company/uk/{id}"
+      "api_list": "/api/company/uk/",
+      "api_detail": "/api/company/uk/{id}/"
```
Comment (roman-tiukh) [inline data_ocean/fixtures/register.json:119]: image

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**